docs/en/rules/Azure.ACR.SoftDelete.md

```diff
@@ -0,0 +1,123 @@
+---
+severity: Important
+pillar: Reliability
+category: Data management
+resource: Azure Container Registry
```

Comment:
Other resources are under `Container Registry`, let's use that. https://azure.github.io/PSRule.Rules.Azure/en/rules/resource/#container-registry

```suggestion
resource: Container Registry
```